Keyfactor/hashicorp-vault-orchestrator
# ab#46718 2.0 Feature Set Complete (#20)

* Kv sub path support (#12)

* updated to support 2 store types. HCVKV and HCV.

* Updated to distinguish between multiple supported store types.

* updated doc to reflect multiple store type configs.

* removed sensitive info from tracelog.

* Removed KEY_SECRET from PutCertificate Task

* Modified PutCertificate to include ---BEGIN ----END banners (for key+cert)

* Allows Cert Stores to be allow a boolean value for SubfolderInventory. Allows inventory to be done on a root storepath and all of the component/subdirectories. Included documentation update to README.md

* Adding InputValidation for KeyValue secrets

* Update generated README

* Include pem chain (#17)

* Fixed store path and mount point mapping

* Fixed issue with path not being resolved before attempting to write cert.

* Allows for Recursive subfolder inventory

* Update readme to call out cert store limits

* Adding InputValidation for KeyValue secrets

* including certificate chain when enrolling via platform.

* added flag on store type to indicate whether to include cert chain

* fixed issue when checking for revocation time for inventory.

* Combine chain certs (#19)

* updated documentation to store Vault token as a secret instead of plain string.

* Updated property names.  now including full chain in ca_chain field.

* fixed issue with revocation time check.

* updated fields.  the full chain is now stored in certificate if selected.

* Fixed issue where improperly formatted secrets would cause inventory to bail.

* Update changelog version info

* updated integration manifest

* Update generated README

* Create keyfactor-merge-store-types.yml

---------
